admin/client/src/components/FieldHolder/OptionFieldHolder.js

```diff
@@ -0,0 +1,28 @@
+import fieldHolder from './FieldHolder';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+
+function optionFieldHolder(Field) {
```

Comment:
Oops, this was a prototype for `CheckboxField.js`, which I thought had deleted/stopped from committing... Will remove it now.